Jenkinsfile.integration

```diff
@@ -3,7 +3,20 @@
 
 def compute_node='storage-rookcheck'
 // the prefix we want to use for the rookcheck environment
-def cluster_prefix="${currentBuild.fullProjectName.replace('/', '_')}-${currentBuild.number}_"
```

Comment:
would it be enough to just use `currentBuild.projectName` instead of `FullProjectName`? Or would that still be to long?